infra/gcp/terraform/k8s-infra-prow-build/prow-build/resources/kubernetes-external-secrets/kubernetes-external-secrets.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kubernetes-external-secrets-metrics
+  namespace: kubernetes-external-secrets
+  labels:
+    app: kubernetes-external-secrets
+spec:
+  selector:
+    app: kubernetes-external-secrets
+  ports:
+    - protocol: TCP
+      port: 3001
+      name: prometheus
+      targetPort: prometheus
+  loadBalancerIP: 34.134.82.56
+  type: LoadBalancer
```

Comment:
A TODO comment to remove it when we have start to have proper monitoring on kubernetes-public.

Reply:
hm this is also a stale IP I think...